include/blast/math/reference/Ger.hpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2019-2024 Mikhail Katliar All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+#pragma once
+
+#include <blast/math/typetraits/MatrixPointer.hpp>
+#include <blast/math/typetraits/VectorPointer.hpp>
+#include <blast/util/Types.hpp>
+
+
+namespace blast :: reference
```

Comment:
I wouldn't go for the name 'reference' like in BLASFEO. I would just call it 'scalar' or so.

Reply:
It is the reference in the sense that it is the implementation which we assume to be correct and use it in the tests as the ground truth. Why do you think this is not a good name?